### ISSUE TITLE
Notification sound

### ISSUE THREAD

**Issue by cakeslice** [ghinda/nudgeti#3]
Suggestion: add the option of adding a notification sound

Example:

```
createNotification();
audioNotification();

function audioNotification(){
    var yourSound = new Audio('yourSound.mp3');
    yourSound.play();
}

function createNotification(){
    var opt = {type: "basic",title: "Your Title",message: "Your message",iconUrl: "your_icon.png"}
    chrome.notifications.create("notificationName",opt,function(){});

    //include this line if you want to clear the notification after 5 seconds
    setTimeout(function(){chrome.notifications.clear("notificationName",function(){});},5000);
}
```

**Comment by ghinda** [comment]
I think this would be a bit annoying, because it wouldn't respect the way you configured your os.

But, if you could create the feature with a setting that is Off by default, I'd love to integrate the pull request.

**Comment by ghinda** [comment]
Available in `0.4` thanks to @cakeslice!
